.github/workflows/nf-core_eager.yml

```diff
@@ -3,10 +3,17 @@ name: nf-core eager CI
 on: [push, pull_request]  
 
 jobs:
+  conda_build:
+    runs-on: ubuntu-latest
```

Comment:
With the little experience I have in this field, I've found that people seem to have a preference for `CentOS/Fedora` on the HPC and workstations.

Wouldn't it be better to rely upon `Ubuntu-18.04 LTS` or perhaps the sturdy `CentOS-7` for the workflow testing?

Reply:
Good point. I forgot to update the OS from @apeltzer 's example script. AFAIK most people in aDNA use Ubuntu so I'll switch to 18.04. But if you know if others using other things let us know and we can set up another workflow.

Reply:
I think, if it's not too much work then perhaps `CentOS-7` could be added as well since it's what `conda-forge` uses to test all it's packages for various reasons. But of course, `Ubuntu` is more user friendly, the `LTS` variant being `DevOps` friendly :)